src/ngraph/ngraph_imperative.cc

```diff
@@ -87,68 +87,125 @@ void NGImperative::parse_ngraph() {
     }
 }
 
+bool compute_forward_imperative(const nnvm::NodeAttrs &attrs,
```

Comment:
Kudos on making it a utility

Reply:
It would be nice have a comment above the function that states:
* The meaning of the return value, and
* (maybe) any noteworthy side-effects this function has.

Reply:
yep, will add comments for clarification.